src/action_install.rs

```diff
@@ -228,11 +228,26 @@ pub fn check_tars_and_move(name: &str, dirs: &RuaDirs, archive_whitelist: &Index
 	});
 
 	for (file, file_name) in dir_items {
-		fs::rename(&file.path(), checked_tars_dir.join(file_name)).unwrap_or_else(|e| {
-			panic!(
-				"Failed to move {:?} (build artifact) to {:?}, {}",
-				&file, &checked_tars_dir, e,
-			)
-		});
+		let src = &file.path();
+		let dst = &checked_tars_dir.join(file_name);
+
+		fs::rename(src, dst)
+			.or_else(|err| {
+				const EXDEV: i32 = 18; //Invalid cross-device link
```

Comment:
_(Err, you seem to have deleted your comment while I was typing a response, but let me address your concerns anyway :stuck_out_tongue:)_

I felt that pulling in the whole [`libc`](https://lib.rs/libc) for a single stable numeric value would be an overkill, but your comment made me reconsider.  
It seems that `rua` depends on `libc` transitively anyway, so I guess we could use [`libc::EXDEV`](https://docs.rs/libc/0.2.68/libc/constant.EXDEV.html) here instead?

Reply:
(sorry about deletion, it was by far past midnight for me, so I felt hesitant whether I'm missing some simple alternative here)

On the libc question. Yes, since we already have `libc` in, I'd say we should use it. Can you change that part please?

Reply:
Terribly sorry for the late response! I've been feeling somewhat under the weather.

_(And no worries, that's perfectly understandable! They were very much valid remarks, though, which why I decided to answer them anyway.)_

Wilco!